### PR TITLE
Added additional test for session security values in php ini

### PR DESCRIPTION
**Pull request by sedninja**
This PR would address Issue #7779 by adding a test for the ini values before trying to set them.

### CODE REVIEW

File: src/Network/Session.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ public static function create($sessionConfig = [])
             }
         }
 
-        if (!isset($sessionConfig['ini']['session.cookie_secure']) && env('HTTPS')) {
+        if (!isset($sessionConfig['ini']['session.cookie_secure']) && env('HTTPS') && ini_get("session.cookie_secure") != 1) {
```
Comment (dereuromark) [inline src/Network/Session.php]: Can we be more strict here using `===`?
Does ini_get() return a string here?

Comment (lorenzo) [inline src/Network/Session.php]: It often returns a string

Comment (sedninja) [inline src/Network/Session.php]: It seems like this depends on how php is configured. Boolean values of "on" and "off" may be returned as 1 and 0 (or an empty string) respectively while true and false may return actual true/false booleans. more strict testing may cause some configurations to fail.

#### http://php.net/manual/en/function.ini-get.php

*Note: When querying boolean values*
A boolean ini value of off will be returned as an empty string or "0" while a boolean ini value of on will be returned as "1". The function can also return the literal string of INI value.

**Comment by markstory** [comment]
Seems fine. I don't think this is the kind of thing we can really have a test for sadly.

**Pull request merged**